airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/manifest.yaml

```diff
@@ -0,0 +1,311 @@
+  retriever_for_type_list:
+    $ref: "#/definitions/retriever_base"
+    record_selector:
+      $ref: "#/definitions/selector"
+      extractor:
+        type: DpathExtractor
+        field_path: []
+  retriever_for_type_list_no_pagination:
+    $ref: "#/definitions/retriever_for_type_list"
+    paginator:
+      type: NoPagination
+
+  # BASE STREAMS
+  base_stream:
+    primary_key: "id"
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/retriever_base"
+  # FULL-REFRESH
+  base_stream_with_list_response_no_pagination:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever_for_type_list_no_pagination"
+  base_stream_with_id_offset_pagination:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever_for_type_list"
+      paginator:
+        $ref: "#/definitions/paginator_id_offset"
+  base_stream_with_time_offset_pagination:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever_base"
+      paginator:
+        $ref: "#/definitions/paginator_time_offset"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          $ref: "#/definitions/requester/request_parameters"
+          # add `fields=<stream_name>(*)` to the request_params
+          fields: "{{ parameters['name'] + '(*)' }}"
+
+  # INCREMENTAL
+  base_incremental_id_stream:
+    $ref: "#/definitions/base_stream_with_id_offset_pagination"
+    retriever:
+      $ref: "#/definitions/base_stream_with_id_offset_pagination/retriever"
+      # this is needed to ignore additional params for incremental syncs
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+    incremental_sync:
+      type: CustomIncrementalSync
+      class_name: source_zendesk_chat.components.id_incremental_cursor.ZendeskChatIdIncrementalCursor
+      cursor_field: "id"
+      field_name: "since_id"
+  base_incremental_time_stream:
+    $ref: "#/definitions/base_stream_with_time_offset_pagination"
+    retriever:
+      $ref: "#/definitions/base_stream_with_time_offset_pagination/retriever"
+      # this is needed to ignore additional params for incremental syncs
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+    incremental_sync:
+      type: CustomIncrementalSync
+      class_name: source_zendesk_chat.components.timestamp_based_cursor.ZendeskChatTimestampCursor
+      use_microseconds: "{{ parameters['use_microseconds'] }}"
+      cursor_field: "{{ parameters['cursor_field'] }}"
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S.%fZ"
+        - "%Y-%m-%dT%H:%M:%SZ"
+      datetime_format: "%s"
+      start_datetime:
+        datetime: "{{ format_datetime(config['start_date'], '%s') }}"
+      start_time_option:
+        field_name: start_time
+        inject_into: "request_parameter"
+
+  # FULL-REFRESH STREAMS
+  # ACCOUNTS
+  accounts_stream:
```

Comment:
The `Accounts` stream doesn't have the `Pagination` at all, returning the List[dict] in response. Yeah, because of the same `base_stream` it would apply this request param, which is fine to pass across the diff endpoints we have for this source.